app/_components/StrategyAnalytics.tsx

```diff
@@ -82,6 +82,10 @@ const StrategyAnalytics = ({ transactionId, network }: props) => {
 		quotesTableRef.current?.getQuotes();
 	};
 
+	useEffect(() => {
```

Comment:
are we just dangling bits of our GUI off other bits of our GUI?

isn't that going to make it very hard to debug?

Reply:
we can follow an approach where we are invalidating both of the quote and trade queries from a single source but this needs to have tanstack implemented to both

we would need to rewrite the trades query and implement tanstack to quotes in order for this to work. we could do it that way if it sounds better?

Reply:
i feel like the longer we leave it the worse it will get so as i understand yes

@hardyjosh what do you think?

Reply:
yeah @findolor let's move it all to Tanstack, makes sense